test/LibraryManager.IntegrationTest/nuget.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should probably go in the repo root, so that we keep the Nuget.config consistent.

Reply:
Will do

Reply:
Will do